Engines/Wine/Verbs/adobeair/script.js

```diff
@@ -0,0 +1,4 @@
+/**
+* Verb to install Uplay
+* @returns {Wine} Wine object
```

Comment:
What's this supposed to do?

Reply:
Messed up the pull sorry.. Will fix 

Expected to include adobeair in verbs